desktop/core/src/desktop/middleware.py

```diff
@@ -953,3 +954,17 @@ def __call__(self, request):
       request.META['HTTP_X_FORWARDED_FOR'] = request.META['REMOTE_ADDR']
 
     return self.get_response(request)
+
+
+class CacheControlMiddleware:
```

Comment:
If defining like this, then need to sure that this doesnt get cherrypicked for py2 Hue.

Ref: https://docs.djangoproject.com/en/3.2/topics/http/middleware/#upgrading-pre-django-1-10-style-middleware

Reply:
We would need it in py2 Hue as well. Changed the middleware to support py2.